about.html

```diff
@@ -12,62 +12,96 @@
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
     
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
+  <STYLE>A {text-decoration: none;} </STYLE>
+        <div class='container-1'>    
+            <img class="item" src="img/lambda-black.png" alt="img Lambda">
+            <a class='item' href="#">Home</a>
+            <a class='item' href="about.html">About</a>
+            <a class='item' href="#">Products</a>
+            <a class='item' href="#">Blog</a>
+            <a class='item' href="#">Contact</a>    
+        </div>
+
+        <img class="imgJumbo" src="img/jumbo-about.png" alt="img Jumbo">
+  
+  
+  
+<br>  
+  <div class="container-about-page">
```

Comment:
This is the quick and dirty way to make space on the page. For this you should apply the box model in the CSS to layout the page rather than just the `br` tag.